site/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bright-js-framework-docs",
+  "private": true,
+  "description": "Documentation for React Serve",
```

Comment:
@chrisdevereux Should this be 'Bright JS Framework' instead of React-Serve?

Reply:
👍